Jenkinsfile

```
@@ -108,7 +168,7 @@ pipeline {
                 expression { !skipStage() }
             }
             parallel {
-                stage('Functional on CentOS 7') {
+                 stage('Functional on CentOS 8') {
```

This really should be:

```suggestion
                 stage('Functional on EL 8') {
```